src/svcutilcore/tools/scripts/UpdateSvcutilDotXmlSerializerPackage.ps1

```diff
@@ -0,0 +1,119 @@
+$logfile = Join-Path $PSScriptRoot 'UpdateSvcutilDotXmlSerializerPackage.log'
```

Comment:
Can you add comment to complain why we need to modify the package? I worry that after sometime, someone will have hard time to figure out why we do it.